Modules/BookClass.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+class Book {
```

Comment:
- [ ] Kindly remove the `eslint-disable` statement in your code. Eslint helps us to write clean, readable and industry-standard codes. Using `eslint-disable` statements defeats this purpose. 👍

Reply:
Sir but if i remove it makes errors.